troubleshooting/debugging.md

```diff
@@ -730,6 +731,25 @@ Remember that we also have older Golang output plugins for AWS (no longer recomm
 
 Consequently, the AWS team contributed an `auto_retry_requests` config option for each of the core AWS outputs. This will issue an immediate retry for any connection issues. 
 
+#### CloudWatch Broken Connection Issues
+
+As mentioned in the section, [Common Network Errors](#common-network-errors), seeing transinent network errors such as `broken connection` are common and normal and do not indicate log loss.
+
+```
+[error] [http_client] broken connection to <your output destination>
+```
+
+However, if you are seeing a large number of `broken_connection` errors to an AWS `cloudwatch_logs` output plugin, several per second, or around one per chunk flush, the following configuration option may resolve this issue:
```

Comment:
let's link to the case studies section to show a typical error rate to clarify large

Reply:
I'm updating this section and adding a link.